RELEASE-NOTES.txt

```diff
@@ -1,5 +1,9 @@
 14.0
 -----
+* Block Editor: Hide image size selection options when image is a url
```

Comment:
The image size selector is still under dev flag, so this should not affect the release build.